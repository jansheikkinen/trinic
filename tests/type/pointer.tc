// Pointer

function main(void) -> void do
  let a: *int8 = undefined;
end
