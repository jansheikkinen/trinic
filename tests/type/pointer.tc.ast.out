
### AST GENERATION ###
Declaration FUNCTION  main
  Stmt VARDECL
    Arg IDENTIFIER
      Identifier_Arg a
        Type POINTER
          Type TYPE INT8
    Arg EXPR
      Expr UNDEFINED
